packages/api-electron/index.js

```diff
@@ -13,4 +18,14 @@ module.exports = () => {
       text: msg.options.body
     });
   });
+
+  ipc.on('ssf-new-window', (e, msg) => {
+    const newWindow = new BrowserWindow();
+    newWindow.loadURL(msg.url);
+    windows.push(newWindow);
+    newWindow.on('close', () => {
+      windows.splice(windows.indexOf(newWindow), 1);
```

Comment:
I think we should only `splice` if `indexOf` returns `> -1` - I know it's an unlikely that it will return `-1` but I think it's worth being defensive here.